servers/triton/samples/simple.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: machinelearning.seldon.io/v1alpha2
+kind: SeldonDeployment
+metadata:
+  name: triton
+spec:
+  protocol: kfserving
+  predictors:
+  - graph:
+      children: []
+      implementation: TRITON_SERVER
+      modelUri: gs://seldon-models/trtis/simple-model
```

Comment:
Does `modelUri` need to follow Triton's model repository layout?

Reply:
I assume so.